build.assets/Makefile

```diff
@@ -65,6 +66,12 @@ endif
 # pass external gocache path through to docker containers
 DOCKERFLAGS := $(DOCKERFLAGS) -v $(GOCACHE):/go/cache -e GOCACHE=/go/cache
 endif
+
+ifeq ("$(RUNTIME_ARCH)","arm64")
+	GRPC_NODE_PLUGIN_BINARY_TYPE := compiled
+else
+	GRPC_NODE_PLUGIN_BINARY_TYPE := prebuilt
+endif
```

Comment:
In Dockerfile-teleterm we need to conditionally execute a bunch of stuff. Instead of prepending each `RUN` with a conditional, I grouped those instructions into separate images and then at the end choose an appropriate image based on the build arg.

https://github.com/gravitational/teleport/blob/a220e2d77d84d9fffc9f2691c3672108c988b21c/build.assets/Dockerfile-teleterm#L28-L53

AFAIK Docker ARG and ENV instructions can't really include any complex logic, so I couldn't encapsulate all of this within the Dockefile. Hence this `ifeq` here.